lib/bibtexrender.php

```diff
@@ -307,9 +307,8 @@ public function printReference($bibtex_key) {
                 // do sth.
                 $pdffilename = mediaFN($this->_conf['pdfdir'][0]) . "/" . $bibtex_key . ".pdf";
                 if (file_exists($pdffilename)) {
-                    resolve_mediaid($this->_conf['pdfdir'][0], $pdflinkname, $exists);
                     $formatstring = $formatstring . '&nbsp;<a href="' . 
-                    ml($pdflinkname) . "/" . $bibtex_key . ".pdf" . '">PDF</a>';
+                    ml($this->_conf['pdfdir'][0] . ":" . $bibtex_key . ".pdf") . '">PDF</a>';
```

Comment:
From the way the code is formatted, it looks like this line is a statement on its own. I suggest the following for somewhat clearer code:
```php
if (file_exists($pdffilename)) {
    $pdflink = ml($this->_conf['pdfdir'][0] . ":" . $bibtex_key . ".pdf");
    $formatstring .= '&nbsp;<a href="' . $pdflink . '">PDF</a>';
}
```